src/main/java/seedu/address/model/recipe/Step.java

```diff
@@ -0,0 +1,50 @@
+package seedu.address.model.recipe;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a step in the list of steps to make the Recipe in the recipe book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidSteps(String)},
```

Comment:
Fix: typo here, should be isValidStep:

```suggestion
 * Guarantees: immutable; is valid as declared in {@link #isValidStep(String)},
```